archive/display/src/main/scala/uk/ac/wellcome/platform/archive/display/DisplayIngestType.scala

```diff
@@ -0,0 +1,36 @@
+package uk.ac.wellcome.platform.archive.display
+
+import io.circe.CursorOp.DownField
+import io.circe.{Decoder, DecodingFailure, Encoder, Json}
+
+sealed trait DisplayIngestType { val id: String }
+
+object CreateDisplayIngestType extends DisplayIngestType {
```

Comment:
We already use `create` as the prefix for a bunch of our generator methods, and the overlap is confusing me as I read this diff. It's a little icky, but would you consider `CreationDisplayIngestType` to disambiguate them?

Reply:
mm I see what you mean, but the id is "create" so I'd prefer to name it as such..